src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPService.kt

```diff
@@ -25,27 +26,31 @@ class CSIPService(@Qualifier("csipApiWebClient") private val webClient: WebClien
     webClient.post()
       .uri("/prisoners/{offenderNo}/csip-records", offenderNo)
       .bodyValue(csipReport)
+      .header("Source", "NOMIS")
```

Comment:
Is this a common pattern we use? Only problem with headers is that they aren't logged anywhere, so not sure how we'd trace the header values (presuming they can be different).

Reply:
Apparently that's how the move and improve use it - I copied the pattern that Andy uses for his.